### DISCUSSION TITLE
Show ERC-20 token balance of the user's choosing

### DISCUSSION THREAD

**Discussion by conradstephens** [rainbow-me/rainbowkit#409] [Ideas]
I love this library it is fantastic and easy to use in many ways. It would also be great if user's can pick the ERC-20 token of their choosing to show its balance when the wallet connects. Please correct me if I am wrong, but at the moment it looks like by default the library shows the balance of the native token of the chain by default with no simple way to change it. Let me know what you all think :)

**Comment by owieth** [comment]
@conradstephens Would love that feature 🚀

**Comment by slavanga** [comment]
Just stumbled upon this requirement and would be very happy to see this implemented.

**Comment by jack-ox** [comment]
I am a power use of Rainbow and this feature would make my life much easier! Given that Metamask has shipped their portfolio feature, if Rainbow does not implement this, I expect more users to flock back to Metamask or other wallets.

**Comment by owieth** [comment]
I've implemented the feature, PR is open here: https://github.com/rainbow-me/rainbowkit/pull/801
